Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -162,6 +162,30 @@ public class NetworkManager : MonoBehaviour
         //    during FinishLoadScene.
         public NetworkManagerMode mode { get; private set; }
 
+        // events
+        public Action onStartClient;
```

Comment:
Can we make these UnityEvents instead so that people can add their handlers in the inspector?